trunk/src/app/srs_app_rtc_source.cpp

```diff
@@ -1266,12 +1263,12 @@ srs_error_t SrsRtcRtpBuilder::package_nalus(SrsSharedPtrMessage* msg, const vect
         pkt->nalu_type = (SrsAvcNaluType)first_nalu_type;
         pkt->header.set_sequence(video_sequence++);
         pkt->header.set_timestamp(msg->timestamp * 90);
-        pkt->set_payload(raw, SrsRtspPacketPayloadTypeNALU);
+        pkt->set_payload(raw_raw, SrsRtspPacketPayloadTypeNALU);
         pkt->wrap(msg);
     } else {
         // We must free it, should never use RTP packets to free it,
         // because more than one RTP packet will refer to it.
-        SrsAutoFree(SrsRtpRawNALUs, raw);
+        SrsUniquePtr<SrsRtpRawNALUs> raw(raw_raw);
```

Comment:
I would rather use a `srs_freep(raw_raw)` manually in this `else` scope to simple the code.

Reply:
Nop, it's used later:

```cpp
        SrsUniquePtr<SrsRtpRawNALUs> raw(raw_raw);
        uint8_t header = raw->skip_first_byte();
```

Won't fix.